apps/docs/src/about/team.md

```diff
@@ -1,72 +1,128 @@
+---
+layout: page
+---
+
 <script lang="ts" setup>
-import { VPTeamMembers } from 'vitepress/theme'
+import { data } from "../index.data";
+import {
+  VPTeamPage,
+  VPTeamPageTitle,
+  VPTeamMembers,
+  VPTeamPageSection
+} from 'vitepress/theme'
+
+const shuffleArray = (array) =>
+  array
+    .map((value) => ({ value, sort: Math.random() }))
+    .sort((a, b) => a.sort - b.sort)
+    .map(({ value }) => value);
 
+    
 // https://vitepress.dev/reference/default-theme-team-page#show-team-members-in-a-page
-const members = [
+// Add your full name here, if you want it to be shown
+const overrides = [
   {
-    avatar: 'https://www.github.com/mj-hof.png',
+    login: "mj-hof",
     name: 'Martin Hofmann',
+    core: true,
     title: 'Product Owner',
-    links: [
-      { icon: 'github', link: 'https://github.com/mj-hof' },
-    ]
   },
   {
-    avatar: 'https://www.github.com/jannick-ux.png',
+    login: "jannick-ux",
     name: 'Jannick Keller',
+    core: true,
```

Comment:
Since you are the "Engineering Lead", would it be more unified if Jannick is the "Design Lead" instead of "Lead Designer"?